### PR TITLE
create .gitattributes

### PR DESCRIPTION
**Pull request by afragen**
How about adding a .gitattributes file so including the project in composer can not include all the non-essential files?

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,10 @@
+* text=auto
+
+.gitattributes export-ignore
+.gitignore export-ignore
+composer.lock export-ignore
+.travis.yml export-ignore
+/ci export-ignore
+/tests export-ignore
+phpunit.xml.dist export-ignore
```
Comment (schlessera) [inline .gitattributes]: The PHPUnit config is in there twice, was probably meant to add PHPCS instead.
```suggestion
phpcs.xml.dist export-ignore
```

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,10 @@
+* text=auto
+
+.gitattributes export-ignore
+.gitignore export-ignore
+composer.lock export-ignore
```
Comment (schlessera) [inline .gitattributes]: This is already excluded by `.gitignore`, so it shouldn't exist in the repository in the first place.
```suggestion
```

File: .gitattributes (hunk 1 above)
Comment (afragen) [reply .gitattributes]: Oops.

**Comment by afragen** [comment]
Thanks for considering this.

**Pull request merged**